### DISCUSSION TITLE
Release 4.2.0 ?

### DISCUSSION THREAD

**Discussion by manolitto** [marcwrobel/jbanking#371] [General]
Hi there,
Your library is great and your RandomXxx tool classes are super-convenient.
Unfortunately the RandomBic class is not yet included in 4.1.0.
Any concrete plans (or roadmap) when version 4.2.0 officially will get released?
Thanks
Manfred

**Comment by marcwrobel** [comment]
Hi @manolitto , I was planning to add a few more features before releasing a new version. But lf you're interested in having one earlier, I will do a release this week.

**Comment by manolitto** [comment]
Thanks for your quick answer @marcwrobel, in the meantime I use a workaround for random BICs. So its not that urgent. However, good to know that a new version is already planned. Looking forward to it.

**Reply by marcwrobel** [reply]
Thanks for the interest, I just released 4.2.0 and it's available [on Maven Central](https://central.sonatype.com/artifact/fr.marcwrobel/jbanking/4.2.0). The changelog is visible on https://github.com/marcwrobel/jbanking/releases/tag/v4.2.0.

I'm eager to hear your feedback on this new version. Please let me know if you have any questions, suggestions, or bug reports.
